Completion/Unix/Command/_zfs

```diff
@@ -185,6 +185,7 @@ _zfs() {
 		$share_rw_properties
 	)
 
+
```

Comment:
not sure if this was intentional but I don't think the extra blank line is needed.

Reply:
it wasn't - actually I originally appended it but I swapped it to use an inline expression because I wasn't sure if order was important.  I'll fix this line when I swap back to appending it